pinotdb/db.py

```diff
@@ -47,6 +48,7 @@ def connect_async(*args, **kwargs):
 def check_closed(f):
     """Decorator that checks if connection/cursor is closed."""
 
+    @wraps(f)
```

Comment:
ahh, i see what you are saying. good catch

Reply:
:-)